bin/hdfcoinc/pycbc_make_coinc_search_workflow

```diff
@@ -42,26 +44,42 @@ parser.add_argument("-d", "--output-dir", default=None,
 wf.add_workflow_command_line_group(parser)
 args = parser.parse_args()
 
-logging.basicConfig(format='%(asctime)s:%(levelname)s : %(message)s', 
-                    level=logging.INFO)
+wf.makedir(args.output_dir)
 
 container = wf.Workflow(args, args.workflow_name)
 workflow = wf.Workflow(args, 'main')
 finalize_workflow = wf.Workflow(args, 'finalization')
 
-wf.makedir(args.output_dir)
 os.chdir(args.output_dir)
 
-rdir = layout.SectionNumber('results', ['configuration',
-                                 'analysis_time',
+rdir = layout.SectionNumber('results', ['analysis_time',
                                  'detector_sensitivity',
                                  'single_triggers',
                                  'coincident_triggers',
                                  'injections',
                                  'search_sensitivity',
                                  'result',
+                                 'workflow',
                                  ])
```

Comment:
Is there any reason not to put this under condiguration/workflow? We already have quite a few top-level sections, and I think this fits well under configuration.